redisson/src/main/java/org/redisson/client/handler/RedisChannelInitializer.java

```diff
@@ -155,6 +155,11 @@ private void initSsl(final RedisClientConfig config, Channel ch) throws KeyStore
         }
         
         SSLParameters sslParams = new SSLParameters();
+
+        if (config.getTlsVersion() != null && !config.getTlsVersion().getTlsVersion().isEmpty()) {
+            sslParams.setProtocols(new String[]{config.getTlsVersion().getTlsVersion()});
```

Comment:
This rightfully takes an array, it should be possible to set an array in RedisClientConfig too.